tests/Grid/Action/MassActionTest.php

```diff
@@ -55,14 +47,6 @@ public function testGetTitle()
         $this->assertEquals($title, $this->massAction->getTitle());
     }
 
-    public function testSetCallback()
```

Comment:
@natewiebe13 for any removals like this - this is just because the `testGetX()` functions do the same thing - the testSet() functions were testing against the class properties directly, but according to https://github.com/sebastianbergmann/phpunit/issues/3339 - private and protected attributes should not be tested directly.